google/cloud/forseti/notifier/notifiers/cscc_notifier.py

```diff
@@ -187,7 +188,8 @@ def find_inactive_findings(new_findings, findings_in_cscc):
             finding_id = finding_list[0]
             to_be_updated_finding = finding_list[1]
 
-            if finding_id not in new_findings_map:
+            if (finding_id not in new_findings_map and
+                    to_be_updated_finding['state'] == 'ACTIVE'):
```

Comment:
Can you also please update the unit test here?  It should be very straightforward, just add a new data with `INACTIVE` state, and current assertion should still pass without any modification.

https://github.com/forseti-security/forseti-security/blob/dev/tests/notifier/notifiers/cscc_notifier_test.py#L125-L186

Reply:
updated!